indico/modules/events/registration/templates/display/_registration_summary_blocks.html

```diff
@@ -1,7 +1,8 @@
-{% macro render_registration_info(registration, from_management=false) %}
-    {{ template_hook('before-render-registration-info', registration=registration, from_management=from_management) }}
-    {% for section in registration.sections_with_answered_fields if (from_management or
-                                                                     (not section.is_manager_only and not section.is_deleted)) -%}
+{% macro render_registration_info(registration, login_user, from_management=false) %}
+    {{ template_hook('before-render-registration-info', registration=registration, from_management=from_management,
+                     login_user=login_user) }}
+    {% for section in registration.sections_with_answered_fields if (section.can_see(login_user, from_management) and
+                                                                     not section.is_deleted) -%}
```

Comment:
I would change the order of the two criteria, ie `not section.is_deleted and ...` - because the first check is always cheap since it's just a boolean variable, while the other is a method call that can do much more.

FWIW it does not really make a big difference here because deleted sections are not THAT common, so it's more a general suggestion since in other cases it may be more significant.